blackdevelopa/iReporter
# #162598267 Signup and Login endpoint

### What does this PR do?
This PR creates the signup and login endpoint to the database

### Description of task to be completed?
* The signup and login endpoint will return a status code of 201 and 200 irrespectively.
* The database is stored remotely on ElephantSQL

### How should this be manually tested?
* Git clone to localhost
* Launch Postman to localhost:3000/auth/signup
* Launch Postman to localhost:3000/auth/login
* Ensure it's on the POST method and click send.
* The required inputs are:
1. username
2. password

### Any background context you want to provide?
There are certain limitations to the get all endpoint. Here's what you should know:
1. This PR does not take into consideration other endpoints.


### What are the relevant pivotal tracker stories?
#162598267

![screen shot 2018-12-12 at 2 43 57 pm](https://user-images.githubusercontent.com/29962968/49875190-b1313200-fe20-11e8-9cd2-23d29afffc5c.png)
